docs/staff/editor/universe.md

```diff
@@ -0,0 +1,176 @@
+---
+sidebar_position: 2
+---
+
+# Universe
+
+The universe is where modules are displayed in a way that students can follow a certain direction or track.
+The role of the editor is to create these universes.
```

Comment:
Is dit niet een gegeven omdat dit in de editor docs zit?

Reply:
True, ik haal het hier wel uit